<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dsReader</class>
 <widget class="QWidget" name="dsReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="all_RB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>all</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="red_RB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>update</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="green_RB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>latest</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="update_B">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="explorer_B">
       <property name="text">
        <string>explorer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doIt3_B">
       <property name="text">
        <string>doSomething</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>457</width>
       <height>272</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>75</width>
       <height>75</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::NoPen</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>75</number>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>icon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ver</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>pathStart</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>update_B</sender>
   <signal>clicked()</signal>
   <receiver>dsReader</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
